npm/webpack-dev-server/src/makeDefaultWebpackConfig.ts

```diff
@@ -78,7 +78,7 @@ export function makeCypressWebpackConfig (
     },
     plugins: [
       new (HtmlWebpackPlugin as typeof import('html-webpack-plugin-5'))({
-        template: indexHtmlFile,
+        template: path.join(projectRoot, indexHtmlFile),
```

Comment:
This is the fix.

Reply:
Some tests are failing on this line: https://app.circleci.com/pipelines/github/cypress-io/cypress/54632/workflows/e04d8993-cb44-42a9-9061-e7b37bef32f2/jobs/2257912

Looks like they are missing setting the `indexHtmlFile` value

Reply:
I think I fixed this in https://github.com/cypress-io/cypress/pull/27320/commits/b557e6e722f3fd1851c8540457146ecfdd6239fd. If `indexHtmlFile` is undefined, we'll just pass undefined like we've been doing. Otherwise we'll call `join` to create the path